src/components/SearchBar.vue

```diff
@@ -5,6 +5,7 @@
         <input v-model="currentSearch"
             type="text"
             placeholder="Search..."
+            :value="defaultTerm"
```

Comment:
I believe this line is unnecessary since v-model is overriding it, and you are updating currentSearch on created anyways